mediator/med_internalstate_mod.F90

```diff
@@ -95,7 +98,8 @@ module med_internalstate_mod
          'glc2ocn_liq ',&
          'fillv_bilnr ',&
          'bilnr_nstod ',&
-         'consf_aofrac'/)
+         'consf_aofrac',&
+         'fillv_consf'/)
```

Comment:
The length of all constant strings needs to be the same,
please pad the fillv_consf variable with a ' ' at the end.